airbyte-webapp/src/locales/en.json

```diff
@@ -269,17 +269,23 @@
   "sources.cancelled": "Cancelled",
   "sources.incomplete": "Retrying",
   "sources.partialSuccess": "Partial Success",
-  "sources.jobStatus.reset_connection.failed": "Reset Failed ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.reset_connection.running": "Reset Running ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.reset_connection.succeeded": "Reset Succeeded ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.reset_connection.cancelled": "Reset Cancelled ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.reset_connection.partialSuccess": "Reset Partial Success ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.sync.failed": "Sync Failed",
-  "sources.jobStatus.sync.running": "Sync Running",
-  "sources.jobStatus.sync.succeeded": "Sync Succeeded",
-  "sources.jobStatus.sync.cancelled": "Sync Cancelled",
-  "sources.jobStatus.sync.partialSuccess": "Sync Partial Success",
-  "sources.jobStatus.unknown": "Status Unknown",
+  "jobs.jobStatus.reset_connection.failed": "Reset Failed ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.reset_connection.running": "Reset Running ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.reset_connection.succeeded": "Reset Succeeded ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.reset_connection.cancelled": "Reset Cancelled ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.reset_connection.partialSuccess": "Reset Partial Success ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.sync.failed": "Sync Failed",
+  "jobs.jobStatus.sync.running": "Sync Running",
+  "jobs.jobStatus.sync.succeeded": "Sync Succeeded",
+  "jobs.jobStatus.sync.cancelled": "Sync Cancelled",
+  "jobs.jobStatus.sync.partialSuccess": "Sync Partial Success",
+  "jobs.jobStatus.check_connection_source.failed": "Configuration check failed",
+  "jobs.jobStatus.check_connection_source.succeeded": "Configuration check finished",
```

Comment:
:information_source: I tried to avoid calling it "succeeded" here for the connector check jobs, since the actual check inside that job might still determined the connection would fail (as shown in the screenshot in the PR description). Thus I felt "finished" would be the better term for those two types of jobs.